web/src/settings_invites.js

```diff
@@ -213,7 +296,85 @@ export function on_load_success(invites_data, initialize_event_handlers) {
         $(".dialog_submit_button").attr("data-is-multiuse", meta.is_multiuse);
     });
 
-    $(".admin_invites_table").on("click", ".resend", (e) => {
+    $(".admin_invites_table").on("click", ".edit-invite", (e) => {
+        // This click event must not get propagated to parent container otherwise the modal
+        // will not show up because of a call to `close_active_modal` in `settings.js`.
+        e.stopPropagation();
+        e.preventDefault();
+
+        const invite_id = Number.parseInt(e.currentTarget.dataset.inviteId, 10);
+        const invite = meta.invites.find((invite) => invite.id === invite_id && invite.is_multiuse);
```

Comment:
Though this works, but we can maybe do it by attaching a `data-invite-id` attribute to the row element. We also do this for the users and bots list, so would be nice to have some consistency.